src/main/java/com/gmail/yuramitryahin/model/Ticket.java

```diff
@@ -0,0 +1,58 @@
+package com.gmail.yuramitryahin.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "ticket")
+public class Ticket {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @OneToOne
```

Comment:
are you sure that here should be 1-1 relationship?

Reply:
oops